packages/aws-cdk-lib/aws-dynamodb/README.md

```diff
@@ -1,6 +1,704 @@
+      region: 'us-east-1',
+      globalSecondaryIndexOptions: {
+        gsi1: {
+          readCapacity: dynamodb.Capacity.autoscaled({ minCapacity: 1, maxCapacity: 10 })
+        },
+      },
+    },
+    {
+      region: 'us-east-2',
+      globalSecondaryIndexOptions: {
+        gsi2: {
+          contributorInsights: false,
+        },
+      },
+    },
+  ],
+});
+```
+
+Note: `contributorInsights` for `globalSecondaryIndexes` are inherited from the `GlobalTable`.
+
+## Local Secondary Indexes
+
+A `GlobalTable` can be configured with `localSecondaryIndexes` when the `GlobalTable` also has a `sortKey`:
+
+```ts
+const globalTable = new dynamodb.GlobalTable(this, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  sortKey: { name: 'sk', type: dynamodb.AttributeType.NUMBER },
+  localSecondaryIndexes: [
+    {
+      indexName: 'lsi',
+      sortKey: { name: 'sk', type: dynamodb.AttributeType.NUMBER },
+    },
+  ],
+});
+```
+
+Alternatively, you can add a `localSecondaryIndex` using the `addLocalSecondaryIndex` method:
+
+```ts
+const globalTable = new dynamodb.GlobalTable(this, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  sortKey: { name: 'sk', type: dynamodb.AttributeType.NUMBER },
+  localSecondaryIndexes: [
+    {
+      indexName: 'lsi1',
+      sortKey: { name: 'sk', type: dynamodb.AttributeType.NUMBER },
+    },
+  ],
+});
+
+globalTable.addLocalSecondaryIndex({
+  indexName: 'lsi2',
+  sortKey: { name: 'sk', type: dynamodb.AttributeType.NUMBER },
+});
+```
+
+## Tags
+
+You can apply `tags` to a `GlobalTable` and any replicas. `Tags` on the `GlobalTable` will only apply to the table in the main deployment region.
+
+```ts
+import * as cdk from 'aws-cdk-lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'Stack', { env: { region: 'us-west-2' } });
+
+const globalTable = new dynamodb.GlobalTable(stack, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  // tags will only apply to table in us-west-2
+  tags: [
+    { key: 'foo', value: 'bar' },
+    { key: 'fizz', value: 'buzz' },
+  ],
+  replicas: [
+    { region: 'us-east-1' },
+    { region: 'us-east-2' },
+  ],
+});
+```
+
+Replicas will not inherit tags from the `GlobalTable`, bou can apply tags on a per-replica basis:
+
+```ts
+import * as cdk from 'aws-cdk-lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'Stack', { env: { region: 'us-west-2' } });
+
+const globalTable = new dynamodb.GlobalTable(stack, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  tags: [
+    { key: 'foo', value: 'bar' },
+    { key: 'fizz', value: 'buzz' },
+  ],
+  replicas: [
+    {
+      region: 'us-east-1',
+      tags: [{ key: 'foo', value: 'bar' }],
+    },
+    {
+      region: 'us-east-2',
+      tags: [{ key: 'fizz', value: 'buzz' }],
+    },
+  ],
+});
+```
+
+## Importing Existing Global Tables
```

Comment:
We don't say "importing" anymore. Change this into "referencing" (and also below)

Reply:
This can go quite far below.

Reply:
Updated.